### DISCUSSION TITLE
ERROR    Uncaught exception: No training images were found for NeRF training!

### DISCUSSION THREAD

**Discussion by Zastyun** [NVlabs/instant-ngp#627] [Q&A]
20:03:30 ERROR    Uncaught exception: No training images were found for NeRF training!
do I need to do the demo first? I don't know how i would fix this

**Comment by acemi159** [comment]
Hmm, I also had the same error and the problem was that the generated `transforms.json` by running colmap, the pipeline passes the image paths to transforms.json via full path and during training network searches for this full image path with respect to the script's path. So the issue is about relative path and full path. run colmap generates full image path and run.py looks for relative paths. So if you just remove the redundant part of the path from the generated transforms.json file for each image then it should solve the problem
